### ISSUE TITLE
How to update from MineOS python based to node.js?

### ISSUE THREAD

**Issue by HAlexer** [hexparrot/mineos#130]
Hi, the NodeJS version is really better than python based, so how to update to node? Servers config files are the same? Thank you.

**Comment by hexparrot** [comment]
There is no upgrade path mostly because the installation path for any of the web-uis are so platform-dependent.  The instructions to systematically remove the python webui depend specifically on the method you used to install it, then following that, you have to use the corresponding installation steps to get the new webui.

It isn't possible to write up such instructions without introducing tons of room for user-error, which is likely to be far more confusing and cumbersome than a fresh install would be, with the OS and all components current and built clean.

You can, however, take archives created from your old setup and import them using the new webui.

TL;DR it is possible to remove all the components of the Python webui, but my experience with various users attempting this typically end up doing a new install instead.

**Comment by HAlexer** [comment]
Done and work fine thank you
